openmdao/test/sellar.py

```diff
@@ -142,8 +142,10 @@ def __init__(self):
         self.add('px', IndepVarComp('x', 1.0), promotes=['*'])
         self.add('pz', IndepVarComp('z', np.array([5.0, 2.0])), promotes=['*'])
 
-        self.add('d1', SellarDis1withDerivatives(), promotes=['*'])
-        self.add('d2', SellarDis2withDerivatives(), promotes=['*'])
+        sub = self.add('sub', Group(), promotes=['*'])
+        sub.ln_solver = ScipyGMRES()
+        sub.add('d1', SellarDis1withDerivatives(), promotes=['*'])
+        sub.add('d2', SellarDis2withDerivatives(), promotes=['*'])
```

Comment:
So with these changes, we have two redundant models now (SellarNoDerivatives and SellarDerivativesGrouped). We talked about leaving this test -- SellarDerivatives -- in the not-recommended original state where the MDA is up in the top group with components that aren't in the MDA (the obj and con) just so that we have some tests that cover the not-recommended configuration (else, we should give an error when they aren't optimal.)